tools/code_format/check_format.py

```diff
@@ -415,9 +417,26 @@ def reportError(message):
       # If we see a version marker we are now in the section for the current release.
       in_current_release = True
 
-    if line.startswith("*") and not VERSION_HISTORY_NEW_LINE_REGEX.match(line):
-      reportError("Version history line malformed. "
-                  "Does not match VERSION_HISTORY_NEW_LINE_REGEX in check_format.py\n %s" % line)
+    # Do basic alphabetization checks of the first word on the line and the
+    # first word after the :
+    if line.startswith("*"):
+      match = VERSION_HISTORY_NEW_LINE_REGEX.match(line)
+      if not match:
+        reportError("Version history line malformed. "
+                    "Does not match VERSION_HISTORY_NEW_LINE_REGEX in check_format.py\n %s" % line)
+      else:
+        first_word = match.groups()[0]
```

Comment:
Based on the * quantifier in `VERSION_HISTORY_NEW_LINE_REGEX` could this just be `:`? Is that OK? (Alphabetization makes sense of it, but just as a release note)

Reply:
I guess arguably - that'd be a useless release note but we should complain cleanly :-)

I'm doing a follow-up to make sure sentences end with . - OK if I change the * to + there, just for release reasons?

Reply:
Yes, that seems like a better place! LGTM otherwise

Reply:
OK sounds good I will force merge once the docs build passes.